tcms/testruns/tests/test_admin.py

```diff
@@ -47,3 +48,62 @@ def test_superuser_can_delete_testrun(self):
                                             args=[self.test_run.pk]), {'post': 'yes'}, follow=True)
         self.assertRedirects(response, reverse('admin:testruns_testrun_changelist'))
         self.assertEqual(TestRun.objects.filter(pk=self.test_run.pk).exists(), False)
+
+
+class TestTestExecutionStatusAdmin(TestCase):
+
+    def test_admin_testrun_execution_status_idle(self):
+        self.test_exe = TestExecutionStatus.objects.get(pk=1)
+        self.assertEqual(self.test_exe.name, "IDLE")
+        self.assertEqual(self.test_exe.color, "#72767b")
+        self.assertEqual(self.test_exe.icon, "fa fa-question-circle-o")
+        self.assertEqual(self.test_exe.weight, 0)
```

Comment:
All of the test methods in this class are essentially the same scenario. 

Can you define the test steps for me (either 1-2-3 or the given-when-then) ?  I'm not sure I understand what is the intention here ? 

Alternatively if you have a question post it in the issue so it can be answered.